std/typecons.d

```diff
@@ -791,26 +791,47 @@ template Tuple(Specs...)
             import std.format : FormatSpec;
 
             /**
-             * Formats `Tuple` with either `%s`, `%(inner%)` or `%(inner%|sep%)`.
+             * Formats Tuple with either `%s`, `%t`, `%r`, `%(`inner`%)` or `%(`inner`%|`sep`%)`.
              *
              * $(TABLE2 Formats supported by Tuple,
              * $(THEAD Format, Description)
-             * $(TROW $(P `%s`), $(P Format like `Tuple!(types)(elements formatted with %s each)`.))
-             * $(TROW $(P `%(inner%)`), $(P The format `inner` is applied the expanded `Tuple`, so
-             *      it may contain as many formats as the `Tuple` has fields.))
-             * $(TROW $(P `%(inner%|sep%)`), $(P The format `inner` is one format, that is applied
-             *      on all fields of the `Tuple`. The inner format must be compatible to all
-             *      of them.)))
+             * $(TROW $(P `%s`), $(P Format like `Tuple!(`types`)(`elements formatted with `%s` each`)`.))
+             * $(TROW %(P `%t`), $(P Format like `(`elements formatted with `%s` each`)`.))
+             * $(TROW %(P `%r`), $(P Format like `%t`. Nested tuples are formatted like `%t` recursively.))
+             * $(TROW $(P `%(`inner`%)`), $(P The format `inner` is applied the expanded Tuple, so it may
+             *      contain as many formats as the Tuple has fields.))
+             * $(TROW $(P `%(`inner`%|`sep`%)`), $(P The inner format is applied on all fields of the Tuple
+             *      and must be compatible to all of them.)))
+             * For printing simple but user-friendly representations of `Tuple`s, `%r` is recommended.
+             * The behavior of `%s` has not been changed to the one of `%t` or `%r` for compatibility reasons.
+             * `%s` is also kept as it is useful in debugging and unit tests.
              * ---
              *  Tuple!(int, double)[3] tupList = [ tuple(1, 1.0), tuple(2, 4.0), tuple(3, 9.0) ];
```

Comment:
No. `unittest`s in templates create a new test for each instance. Therefore we have DIP82 preventing it. As `static unittest` comes, we will do that (one day).
Note that Tuple is instantiated very often.